src/components/teamBoxes/teamOneBox.css

```diff
@@ -4,17 +4,20 @@
   justify-content: center;
 }
 .redBoxCard {
-  height: fit-content;
+  display: flex;
+  flex-direction: column;
   color: #d2dcde;
   padding: 15px 15px 25px 15px;
-  font-size: 16px;
+  font-size: larger;
```

Comment:
to be honest, not a fan of the enlarged team box text, on my screen (1920x1080)it looks like old people's phones when they set fonts in large
![Screen Shot 2023-02-19 at 3 42 01 PM](https://user-images.githubusercontent.com/96794722/219974202-4493c647-176d-4ec4-91d4-c5871576a2b1.png)

I would suggest taking the font-size rule here and just using their h3 tag and it will look like the below:

![Screen Shot 2023-02-19 at 3 42 06 PM](https://user-images.githubusercontent.com/96794722/219974238-513feca4-a481-4f83-b1d4-bf62ce4121d9.png)